src/api/mock.ts

```diff
@@ -291,10 +304,12 @@ export class MockConnection implements Connection, MockBase {
         };
         items.observers.push(observer);
 
-        return Rx.Observable.just<any>({
+        let observable = Rx.Observable.just<any>({
             observer: observer.observerId,
             items: this._updateMockObserver(observer, items, false),
         });
+        if (this._simulateDelay) observable = observable.delay(100);
```

Comment:
Maybe do `return this._simulateDelay ? observable.delay(100) : observable`? and have `const observable`?